src/core/remote.js

```diff
@@ -1822,6 +1835,10 @@ Remote.prototype.findAccount = function(accountID) {
  */
 Remote.prototype.createPathFind = function(options, callback) {
   if (this._cur_path_find !== null) {
+    if (callback === undefined) {
```

Comment:
I just thought of maybe a better idea: I think we can just skip this whole queuing block if the `callback` is undefined. That would better preserve the old behavior.